ddtrace/contrib/logging/__init__.py

```diff
@@ -15,13 +15,17 @@
 Patch ``logging``
 ~~~~~~~~~~~~~~~~~
 
-If using :ref:`ddtrace-run<ddtracerun>` then set the environment variable ``DD_LOGS_INJECTION=true``.
+There are a few ways to tell ddtrace to patch the ``logging`` module:
 
-Or use :func:`patch()<ddtrace.patch>` to manually enable the integration::
+1. If using :ref:`ddtrace-run<ddtracerun>`, you can set the environment variable ``DD_LOGS_INJECTION=true``.
+
+2. Use :func:`patch()<ddtrace.patch>` to manually enable the integration::
 
     from ddtrace import patch
     patch(logging=True)
 
+3. Set ``log_injection_enabled`` at runtime via the Datadog UI.
```

Comment:
```suggestion
3. (beta) Set ``log_injection_enabled`` at runtime via the Datadog UI. 
```

since we haven't GA'd the feature yet. Also thank you for updating the docs!